processor/src/errors.rs

```diff
@@ -1,6 +1,14 @@
 use super::{AdviceSetError, CodeBlock, Digest, Felt, Word};
+use crate::system::{FMP_MAX, FMP_MIN};
+use core::fmt::{Display, Formatter};
+use vm_core::stack::STACK_TOP_SIZE;
+use vm_core::utils::to_hex;
+use winterfell::math::FieldElement;
 use winterfell::ProverError;
```

Comment:
nit: I'd prefer this arrangement:
```Rust
use super::{
    system::{FMP_MAX, FMP_MIN},
    AdviceSetError, CodeBlock, Digest, Felt, Word,
};
use core::fmt::{Display, Formatter};
use vm_core::{stack::STACK_TOP_SIZE, utils::to_hex};
use winterfell::{math::FieldElement, ProverError};
```

Reply:
is there any tool to perform this transformation?

Reply:
it seems we can change that on `rustfmt`, is this the configuration you prefer?

https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=import#One%5C%3A

Reply:
I'm not sure there is an automated way to do exactly this. Usually, we just try to have a single import per high-level module.

Reply:
This is the documented behavior of the `rustfmt`, is that good enough?

> Merge all imports into a single use statement as long as they have the same visibility.

Reply:
I personally don't mind either way, I just want to have a tool to automate these things for me :)